### PR TITLE
use the split out obsah library

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 ansible
+git+https://github.com/evgeni/obsah.git@obsah#egg=obsah
```
Comment (evgeni) [inline requirements.txt]: this is obviously temporary until https://github.com/theforeman/obsah/pull/1 is merged and we published `obsah` to pypi.

File: tests/test_obal.py [hunk 2]
```diff
@@ -1,95 +0,0 @@
-import pytest
```
Comment (evgeni) [inline tests/test_obal.py]: this file contained only library tests that now live in `obsah`

File: requirements.txt (hunk 1 above)
Comment (ehelms) [reply requirements.txt]: It has been merged.

Comment (ehelms) [reply requirements.txt]: Sorry, I meant to ask if the plan was to update this with a released Obsah before merging or to use from git for some amount of development time.

Comment (evgeni) [reply requirements.txt]: I think the released version should be fine. Which is what I've updated the file now for.

File: requirements.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 ansible
+obsah
```
Comment (ekohl) [inline requirements.txt]: Shouldn't this also be in setup.py?

File: tox.ini [hunk 4]
```diff
@@ -8,6 +8,7 @@ envlist = py27,py36,py37,ansible-lint
 
 [testenv]
 deps =
+    -r requirements.txt
```
Comment (ekohl) [inline tox.ini]: If you include `obsah` in `setup.py` I don't think this is needed.

File: requirements.txt (hunk 3 above)
Comment (evgeni) [reply requirements.txt]: should, yeah

**Comment by evgeni** [comment]
please don't merge before we don't figure out how to deploy obsah in jenkins

**Comment by ekohl** [comment]
Before we merge this, let's do a final release to tag what's in current master as well?

**Comment by evgeni** [comment]
tagged 0.6.0 before this, and also added obsah to jenkins. let's see what we break.

**Pull request merged**